taprpc/taprootassets.proto

```diff
@@ -999,3 +1006,23 @@ message OutPoint {
     */
     uint32 output_index = 2;
 }
+
+message SubscribeReceiveEventsRequest {
+    // Filter receives by a specific address. Leave empty to get all receive
+    // events for all addresses.
+    string filter_addr = 1;
```

Comment:
What about exposing the event ID and/or timestamp to allow users to recv a backlog of events?

Reply:
Yes, we'll do that in part 2 of the series.